### PR TITLE
Add doc for performance benchmark using Kinvolk

### PR DESCRIPTION
**Pull request by allenlsy**
Add doc for performance benchmark using Kinvolk

Signed-off-by: Allen Leigh <allenlsy@gmail.com>

### CODE REVIEW

File: content/docs/guides/ha_scale/performance.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Open Service Mesh Performance"
+description: "Performance benchmark on control plane and data plane sidecar"
+type: docs
+---
+
+# Performance
```
Comment (shashankram) [inline content/docs/guides/ha_scale/performance.md:7]: This isn't a how-to guide, so could you move this to `docs/performance/`

File: content/docs/guides/ha_scale/performance.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+OSM aims to create a high-performance service mesh while providing all essential network management features. The development team benchmarked its performance and has documented the results.
```
Comment (shashankram) [inline content/docs/guides/ha_scale/performance.md:9]:
```suggestion
OSM aims to create a high-performance service mesh while providing all the essential service mesh features. This document describes the control plane and data plane performance of OSM.
```

**Review by shashankram — COMMENTED** [review]
@allenlsy thanks for drafting this doc. I'll take a closer look at the details and see if there's anything more we'd like to document.

File: content/docs/guides/ha_scale/performance.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+* Control plane
+    * Peak time CPU/Memory: the maximum CPU/memory usage during the whole benchmark process.
+    * Stable time CPU/Memory usage: the average CPU/memory usage the control plane when it is off the peak time and gets stable.
+* Data plane sidecar
+    * Peak time CPU/Memory: the maximum CPU/memory usage among all sidecar containers during the whole benchmark process.
+    * Stable time CPU/Memory usage: the average CPU/memory usage of all data plane sidecar containers when the usage becomes stable.
+* Request latency: P50, P90, P99 and maximum of request latencies are collected.
+
+## Process
```
Comment (zr-msft) [inline content/docs/guides/ha_scale/performance.md:29]:
```suggestion
## What we tested
```

**Comment by allenlsy** [comment]
Closing this PR. The doc will be replaced by https://github.com/openservicemesh/osm-docs/pull/342

**Pull request closed without merging**